apps/multiplatform/common/src/androidMain/kotlin/chat/simplex/common/views/chatlist/ChatListNavLinkView.android.kt

```diff
@@ -20,11 +20,11 @@ actual fun ChatListNavLinkLayout(
   disabled: Boolean,
   selectedChat: State<Boolean>,
   nextChatSelected: State<Boolean>,
-  oneHandUI: State<Boolean>
+  chatToolbarOnBottom: State<Boolean>
```

Comment:
not necessary to pass around, can be retrieved as

`ChatModel.controller.appPrefs.chatToolbarOnBottom`

(occurences where we pass these are legacy before we had chatModel globally accessible and were passing it or stuff from it around)